variables.tf

```diff
@@ -123,3 +123,9 @@ variable "tags" {
   description = "A map of tags to add to all resources."
   default     = {}
 }
+
+variable "emr_abac_valid_tags" {
+  type        = map(list(string))
+  description = "A map of valid tags for maintaining EMR resources when using ABAC IAM Policies with Tag Conditions. Make sure your tamr-config.yml file specifies tags key values. Refer to tamr-config module examples for more info."
```

Comment:
The instructions about the tamr configuration is a bit vague. We should either find a way to make this guidance more specific, or we leave it completely up to the configuration module to guide the user on how to use this field.

I'm leaning toward the latter, but open to either.

Reply:
Added examples into configuration module https://github.com/Datatamer/terraform-aws-tamr-config/pull/15